stable/insights-agent/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+appVersion: 0.0.1
+description: A Helm chart to run the Fairwinds Insights agent
+name: insights-agent
+version: 0.4.0
+maintainers:
+  - name: bobby-brennan
```

Comment:
Are you going to be the only maintainer?

Reply:
Added Barnabas